src/controllers/groupe.controller.ts

```diff
@@ -0,0 +1,83 @@
+import { NextFunction, Request, Response } from 'express';
+import { Group } from '@prisma/client';
+import { CreateGroupDto } from '@dtos/group.dto';
+import groupService from '@services/groups.service';
```

Comment:
To adhere to the convention in JavaScript/TypeScript, consider starting class names with a capital letter for clarity.

```suggestion
import GroupsService from '@services/groups.service';
```

Reply:
To ensure consistency and clarity, consider renaming the file from `groupe.controller.ts` to `groups.controller.ts`.